Shopipy.ApiService/Services/AuthService.cs

```diff
@@ -0,0 +1,30 @@
+using System.IdentityModel.Tokens.Jwt;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+using Shopipy.Persistence.Models;
+
+namespace Shopipy.ApiService.Services;
+
+public class AuthService(IConfiguration config)
+{
+    public string GenerateToken(User user)
+    {
+        var handler = new JwtSecurityTokenHandler();
+
+        var securityKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(config["Authentication:Jwt:Key"]!));
```

Comment:
Cache this GetBytes call.

Reply:
Passing `signingCredentials` to service now.